### ISSUE TITLE
bootstrapping a new Dev setup 

### ISSUE THREAD

**Issue by scanlonjts** [iteego/glasir.demo#1]
cloned
./gradlew cleanAll devDeploy
BUILD SUCCESSFUL

Total time: 11 hrs 25 mins 9.853 secs

./gradlew startDevStore failing to start
Caused by: java.lang.ClassNotFoundException: org.jboss.bootstrap.BaseServerConfig

**Comment by mpemer** [comment]
Current version of the demo runs atg 10 and jboss 5.1.2, which in turn means jdk 1.6 is required. Please verify with "java -version".

**Comment by scanlonjts** [comment]
ah - I have jdk 1.7

will you have an updated demo kit at some point - jdk 1.6 is really old and even the 1.7 EOL process has started 

Joe Scanlon

Unauthorized disclosure and/or use of information contained in this email may result in civil and criminal liability

**Comment by mpemer** [comment]
Yes. There will be an atg11 version of the demo within the next month. Atg11 runs on jboss 6, which in turn means we can use jdk 1.7. The glasir platform is agnostic to the jdk version and the only driving factor is the versions of the software that we deploy. Glasir runs on whatever jdk gradle runs on. We look forward to getting support for jdk8 - but need to wait for our friends at Oracle to support it first. :)

**Comment by scanlonjts** [comment]
OK - makes sense -
I will check back periodically to see when you have a v11 ready -
Joe Scanlon

M: 603.459.3242 
F: 1-941-296-7907
E: joe.scanlon@allstar-it.com mailto:joe.scanlon@allstar-it.com

CONFIDENTIALITY NOTICE: This email message (including attachments) contains information which may be confidential and/or legally privileged.
Unless you are the intended recipient, you may not use, copy or disclose to anyone the message or any information contained in the message or from any attachments that were sent with this email, and If you have received this email message in error, please advise the sender by email, and delete the message. 

Unauthorized disclosure and/or use of information contained in this email may result in civil and criminal liability
